pyaedt/application/AnalysisIcepak.py

```diff
@@ -253,7 +254,11 @@ def export_3d_model(self, fileName, filePath, fileFormat=".step", object_list=[]
             allObjects = object_list[:]
 
         self.logger.info("Exporting {} objects".format(len(allObjects)))
-
+        major = -1
+        minor = -1
+        if fileFormat in [".step", ".stp", ".sm3", ".sat", ".sab"]:
+            major = 29
+            minor = 0
```

Comment:
@maxcapodi78 Could you add a short comment here to explain those values? Maybe it is something common and usual for Icepak user.

Reply:
Done